src/Core/System/SalesChannel/Event/SalesChannelIndexerEvent.php

```diff
@@ -9,6 +9,10 @@
 #[Package('discovery')]
 class SalesChannelIndexerEvent extends NestedEvent
 {
+    /**
+     * @param array<array-key, string> $ids
```

Comment:
```suggestion
     * @param array<string> $ids
```
most of the time, we omit the key :man_shrugging: